packages/flutter_markdown/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
```

Comment:
There's no reason to have a separate license file.  It can just use the license file that's already in the root of the repository.